packages/palette/src/elements/DataVis/ChartHoverTooltip.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext, useRef } from "react"
+import styled from "styled-components"
+import { color } from "../../helpers/color"
+import { media } from "../../helpers/media"
+import { Flex } from "../Flex"
+import { MousePositionContext } from "./MousePositionContext"
+
+const LABEL_OFFSET = 10
+
+/**
+ * Tooltip for bar and line charts
+ */
+export const ChartHoverTooltip = ({ children }) => {
```

Comment:
```ts
export const ChartHoverTooltip: React.FC< ChartHoverTooltipProps> = ...
```

Reply:
hmmm I see these different notations of defining component props but don't know what the difference is 🤔

Reply:
The version currently being used is just a normal function with an interface. By assigning `React.FC<Props>` we confer additional type information to the component -- that it is a React component, that it is a Functional component, and so on.

Reply:
got it got it! it is type definition for `ChartHoverTooltip` itself and not the function params!